calcom/cal.com
# feat: cal video no show workflow triggers

## What does this PR do?



- Fixes #XXXX (GitHub issue number)
- Fixes CAL-XXXX (Linear issue number - should be visible at the bottom of the GitHub issue description)



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

1) Go to workflows
2) Create "after hosts didn't join cal video" workflow

Check if task was added in Task Table in prisma studio

you can also execute it by making request to http://localhost:3000/api/tasks/cron
